### ISSUE TITLE
[UWP] FastGridCell.PrepareCell is never called

### ISSUE THREAD

**Issue by deakjahn** [NobsterTheLobster/Xamarin.Forms.GridView#29]
FastGridCell is basically completely useless on UWP.

You might force it in some cases with:

    public MyCell() {
      InitializeComponent();

      if (Device.RuntimePlatform == Device.UWP)
        PrepareCell(new Size(..., ...);
    }

But if `PrepareCell` is not called, `IsInitialized` stays false and even a context change will not call `SetupCell`.

**Comment by NobsterTheLobster** [comment]
Yes that's a fair point. I'm just leveraging Xamarin.Forms own render for UWP so yes it is useless.

This control and the FastGridCell object originated from this repo https://github.com/twintechs/TwinTechsFormsLib. 

I never liked the FastGridCell concept myself since I'm more of XAML guy and also I wasn't fully convinced of its of performance claims. I thought about dropping the FastGridCell entirely when I started this repo but in the end I left it and now there is delema of what to do about UWP.

**Comment by deakjahn** [comment]
Even if you're basically XAML, you can inherit your cells from it and use them (I do it on all platforms, with the UWP caveats above).
